pkgs/applications/office/libreoffice/online/node-packages.nix

```diff
@@ -0,0 +1,4675 @@
+# This file has been generated by node2nix 1.7.0. Do not edit!
```

Comment:
I would prefer this one to be integrated into `pkgs/development/node-packages/node-packages-v10.json`:

We do this for other packages as well, i.e.: 
```
, { "iosevka-build-deps": "../../data/fonts/iosevka" }
```

Thanks!